deployment/roles/reverseproxy/templates/security-headers.toml.j2

```diff
@@ -0,0 +1,35 @@
+[tls.options]
+    [tls.options.default]
+      minVersion = "VersionTLS12"
+      sniStrict = true
+      cipherSuites = [
+          "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256",
+          "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+          "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+          "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
+          "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
+          "TLS_AES_128_GCM_SHA256",
+          "TLS_AES_256_GCM_SHA384",
+          "TLS_CHACHA20_POLY1305_SHA256",
+          "TLS_FALLBACK_SCSV"
+      ]
+
+[http.middlewares]
+    [http.middlewares.security-headers.headers]
+      # CORS
+      AccessControlAllowMethods = ["GET", "OPTIONS", "PUT"]
+      AccessControlAllowOrigin = "origin-list-or-null"
+      AccessControlMaxAge = 100
+      #AddVaryHeader = true
+      BrowserXssFilter = true
+      ContentTypeNosniff = true
+      ForceSTSHeader = true
+      FrameDeny = true
+      SSLRedirect = true
+      STSIncludeSubdomains = true
+      STSPreload = true
+      ContentSecurityPolicy = "default-src 'self' https://validator.spaceapi.io/v1/validate/ https://use.fontawesome.com/releases/v5.8.1/; script-src 'self' 'unsafe-inline'"
+      CustomFrameOptionsValue = "SAMEORIGIN"
+      ReferrerPolicy = "same-origin"
+      FeaturePolicy = "vibrate 'self'"
+      STSSeconds = 315360000
```

Comment:
This record should probably be moved up so it's in the same "block" as the other HSTS headers.

Reply:
Agreed, i've moved it to the rest :)